app/controllers/v1/political_parties_controller.rb

```diff
@@ -23,4 +31,23 @@ def set_party
     @party = PoliticalParty.find_by(id: params[:id])
     render status: :not_found && return unless @party
```

Comment:
It's a `before_action` for all actions that need to set find a political party using `id`. If the party isn't found it renders 404. otherwise it sets it to the `@party` instance variable and continues to the action.